helix-term/src/ui/editor.rs

```diff
@@ -719,7 +741,16 @@ impl Component for EditorView {
 
         for (view, is_focused) in cx.editor.tree.views() {
             let doc = cx.editor.document(view.doc).unwrap();
-            self.render_view(doc, view, area, surface, &cx.editor.theme, is_focused);
+            let loader = cx.editor.syn_loader.clone();
+            self.render_view(
+                doc,
+                view,
+                area,
+                surface,
+                &cx.editor.theme,
+                is_focused,
+                loader,
```

Comment:
Pass in `&cx.editor.syn_loader`, make the method use `loader: &syntax::Loader`. That way we don't need to clone the loader everywhere.